modules/tinymce/src/core/test/ts/browser/EditorManagerTest.ts

```diff
@@ -55,7 +55,7 @@ describe('browser.tinymce.core.EditorManagerTest', () => {
 
         // Re-init on same id
         EditorManager.init({
-          selector: '#' + EditorManager.activeEditor.id,
+          selector: '#' + (EditorManager.activeEditor as Editor).id,
```

Comment:
Here was well. At least there is no error shown when doing a strict type check

Reply:
See previous comment...